main.nf

```diff
@@ -194,10 +194,10 @@ if (workflow.profile.contains('awsbatch')) {
 
 
 // Stage config files
-ch_multiqc_config = file("$baseDir/assets/multiqc_config.yaml", checkIfExists: true)
```

Comment:
There are 3x other instances of `$baseDir` in `main.nf` (L163, 172, 1071) - would be great if you could also update them to `$projectDir`.